interval.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+type RefreshInterval time.Duration
+
+func (self RefreshInterval) String() string {
+	return time.Duration(self).String()
+}
+
+func (self *RefreshInterval) Set(v interface{}) error {
+	switch v := v.(type) {
+	case int:
+		*self = RefreshInterval(time.Duration(v) * time.Second)
+		return nil
+	case string:
+		i, err := strconv.ParseUint(v, 10, 64)
+		if err == nil {
+			*self = RefreshInterval(time.Duration(i) * time.Second)
+			return nil
+		}
+		d, err := time.ParseDuration(v)
+		if err != nil {
+			return err
+		}
+		ii := RefreshInterval(d)
+		if ii < MinimumRefreshInterval {
+			return fmt.Errorf("Refresh interval must be equal orgreater than or equal to %s", MinimumRefreshInterval)
+		}
+		*self = ii
+		return nil
+	default:
+		return fmt.Errorf("Invalid refresh interval: %v", v)
+	}
+}
+
+func (self *RefreshInterval) SetYAML(tag string, v interface{}) bool {
+	err := self.Set(v)
+	return err != nil
```

Comment:
If `self.Set(v)` errors, this should return `false`. I think the opposite is true here. Also, to make it more readable, could we convert this to:

```
if err != nil {		
    return false		
}		
return true
```